src/YoloDev.Expecto.TestSdk/build/net6.0/YoloDev.Expecto.TestSdk.props

```diff
@@ -1,10 +1,26 @@
 <Project>
+
+  <PropertyGroup>
+    <!-- Makes the new platform/runner opt-in feature -->
+    <EnableExpectoRunner Condition=" '$(EnableExpectoRunner)' == '' ">false</EnableExpectoRunner>
```

Comment:
Please let me know if you prefer the new platfrom to be an opt-in feature or opt-out one.

Reply:
Module members may get a + in their name at some point - maybe load this DLL in FSI and check the generated name?

Reply:
This depends on whether or not it's supported "everywhere" - though, I guess what we could do is merge with `false`, release, wait a month or so, and then do a breaking release with `true`.

Reply:
Thanks to the bridge component, everything that was working with VSTest before will continue to work. Having this entry `true` by default will only create the self-contained executable in addition to the old modes. This flag is primarly here as a safetynet in case some unexpected scenario happens so that users can fully opt-out the new mode while we could work on a fix.